projects/notebooks/testing/config.yaml

```diff
@@ -257,7 +259,7 @@ tests:
     # The test flavor to run. Can be: ods-ci, notebook-performance or gating
     test_flavor: ods-ci
     identifier: unspecified
-    minio_bucket_name: notebooks
+    minio_bucket_name: ds-pipeline-dspa
```

Comment:
you need to revert this modification

Reply:
while  RUNNING THIS STEP it is trying to emty notebooks bucket instead of ds-pipeline-dspa  bucket
```shell

``` ~~ projects/notebooks/roles/notebooks_dashboard_scale_test/tasks/main.yml:174
~~ TASK: notebooks_dashboard_scale_test : Empty the Minio S3 bucket
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


<command>
oc -c mc -n notebook-scale-test-minio rsh $(oc get pod -lapp=minio -n notebook-scale-test-minio -oname)   mc --config-dir /tmp rm --insecure --recursive --force --quiet minio/ds-pipeline-dspa/ >/dev/null;
oc -c mc -n notebook-scale-test-minio rsh $(oc get pod -lapp=minio -n notebook-scale-test-minio -oname)   rm -rf /artifacts/to_export > /dev/null
oc -c mc -n notebook-scale-test-minio rsh $(oc get pod -lapp=minio -n notebook-scale-test-minio -oname)   mc --config-dir /tmp cp --insecure /etc/os-release minio/ds-pipeline-dspa; # without it, cp may fail if the bucket is empty

</command>

<stdout> `/etc/os-release` -> `minio/ds-pipeline-dspa/os-release`
<stdout> Total: 0 B, Transferred: 592 B, Speed: 109.51 KiB/s

Reply:
yes, when you run it locally, the preset isn't applied.
so you need to remove it from the commit, but keep it applied in your machine if you want to test locally